src/components/AccountCreation.jsx

```diff
@@ -45,35 +41,35 @@ function AccountCreation() {
       setError('Emails do not match');
     }
 
-    if (
-      !(preferredContact.schoolEmail || preferredContact.secondaryEmail || preferredContact.phone)
-    ) {
-      setError('No preferred contact method selected');
-    }
-
     if (!(paymentMethod.cash || paymentMethod.venmo)) {
       setError('No preferred payment method selected');
     }
 
     try {
-      // setError('');
-      // set up a load state, so when signing up the user, we disabled the "Sign Up" botton below,
-      // so they don't automatically keep clicking the button and create multiple of accounts at the same time
+      setError('');
+      // // set up a load state, so when signing up the user, we disabled the "Sign Up" botton below,
+      // // so they don't automatically keep clicking the button and create multiple of accounts at the same time
       setLoading(true);
-      // await signup(schoolEmailRef.current.value, passwordRef.current.value);
-    } catch {
+      await postCustomer(
+        schoolEmailRef.current.value,
+        passwordRef.current.value,
+        school,
+        preferredContact,
+        secondaryEmailRef?.current?.value ?? null,
+        phoneRef?.current?.value ?? null,
+        paymentMethod,
+        venmoRef?.current?.value ?? null
+      );
+    } catch (e2) {
```

Comment:
```suggestion
    } catch (e) {
```

Reply:
Also related to my other comment, we could print the message of the error that is caught here